dll/shellext/netshell/lang/fr-FR.rc

```diff
@@ -163,7 +163,7 @@ BEGIN
     IDS_NET_DELETE "Supprimer"
     IDS_NET_RENAME "Renommer"
     IDS_NET_PROPERTIES "Propriétés"
-    IDS_NET_OPEN_CONNECTIONS "Open Network Connections"
+    IDS_NET_OPEN_CONNECTIONS "Ouvrir les Connexions réseau"
```

Comment:
Do we want 'C' here, compared to cases on PR8164?

Reply:
@hpoussin had the same question; I consider here that "Connexions réseau" is the title of the shell directory/CPL dialog that's going to be opened. (This IDS_... is what's shown in the network taskbar icon context menu.)

On the contrary, for PR #8164 the "C" -> "c" change in "Options de connexion", is because here I consider "connexion" to simply be a noun naming the object for which the options are being opened.

Reply:
OK I made the change to lowercase "c".